### PR TITLE
Added functionality to support new Pi-Hole teleporter admin function

### PR DESCRIPTION
**Pull request by middleagedman**
I can't quite figure out the best way to fix the debian.docker, and it doesn't seem to use the same php zip extensions.

### CODE REVIEW

**Comment by middleagedman** [comment]
Travis CI build is failing, even though I changed nothing related to debian:jessie.  Is the build env hosed?

test/test_000_build_containers.py::test_build_pihole_image[debian:jessie-debian.docker-diginc/pi-hole:debian] FAILED

**Comment by diginc** [comment]
I think the debian is failing because of changes in 1.12.  I knew when I coded the new installer that I'd have to add a change to pin the versions to make builds reproducible.  I'm pushing that change right now to dev.

install.sh now has a variable called CORE_TAG in it, it builds v1.11.2 just fine but 1.12 breaks the build, so it's nothing you did.  I'll look into this more tomorrow.

**Comment by diginc** [comment]
@middleagedman if you merge dev's latest commit it [fixed the debian build](https://travis-ci.org/diginc/docker-pi-hole/builds/197114698).

File: alpine.docker [hunk 1]
```diff
@@ -25,6 +25,9 @@ RUN apk add --update 'gnupg<2.1.17-r0' && \
 # Customized from submodules
 COPY ./alpine/nginx.conf /etc/nginx/nginx.conf
 
+# Enables Pi-hole Teleporter (extract your lists)
+COPY ./alpine/php.ini /etc/php5/php.ini
```
Comment (middleagedman) [inline alpine.docker]: Hmm.. Maybe.. When I ran the latest alpine docker build, the gui complained that the zip.so extension wasn't loaded..  So yea, you have it done the right way.

Maybe need to make sure the php.ini has
include=/etc/php5/conf.d
?

I'll rebuild it again from master and see what happens.

Comment (middleagedman) [inline alpine.docker]: So yea.. The only change would be to add the "apk add php5-zip" to the install.

**Comment by middleagedman** [comment]
I'll close this merge request.. Since it just messy :)

**Comment by diginc** [comment]
I manually merged your changeset into my local dev branch last night, minus php.ini, so I just pushed that up for #102

**Comment by middleagedman** [comment]
Thanks.. I'm not perfect with GIT, so I appreciate it. :)
I like the Travis CI thing.. That's pretty neat.

**Pull request closed without merging**